actionmailbox/lib/action_mailbox.rb

```diff
@@ -11,6 +11,7 @@ module ActionMailbox
 
   mattr_accessor :ingress
   mattr_accessor :logger
+  mattr_accessor :skip_incineration, default: false
```

Comment:
Can we name this option `incinerate` and make the default value `true`? Then you’d disable incineration by setting it to false.